Unable to Create New React Project Using npm create @pplancq/react-app@latest test-project

**Bug Description:**

There is a major bug in the react-template. It is impossible to create a new React project using the command `npm create @pplancq/react-app@latest test-project`.

**To Reproduce:**

Steps to reproduce the behavior:

1. Run the command `npm create @pplancq/react-app@latest test-project`
2. See error

**Expected behavior:**

The command should successfully create a new React project.

**Additional context:**

Here is the error log:

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: test-project@1.1.0
npm ERR! Found: webpack-dev-server@5.0.3
npm ERR! node_modules/webpack-dev-server
npm ERR!   dev webpack-dev-server@"^5.0.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer webpack-dev-server@"^4.15.1" from @pplancq/webpack-config@1.1.0
npm ERR! node_modules/@pplancq/webpack-config
npm ERR!   dev @pplancq/webpack-config@"^1.0.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

--- a/packages/react-template/package.json
+++ b/packages/react-template/package.json
@@ -75,7 +75,7 @@
     "vitest-sonar-reporter": "^2.0.0",
     "webpack": "^5.90.3",
     "webpack-cli": "^5.1.4",
-    "webpack-dev-server": "^5.0.2"
+    "webpack-dev-server": "^4.15.1"
   },
   "volta": {
     "node": "20.10.0",

